README.md

```diff
@@ -310,7 +310,7 @@ Pass the required parameters to the action by configuring them in the `actions/{
 {action name}:
   function: commerce/{action name}/index.js
   web: 'no'
-  runtime: nodejs:16
+  runtime: nodejs:20
```

Comment:
Could an environment variable be used here?

Reply:
I don't think it is possible. As far as I am concerned env vars can only be used in the `input` section of the `app.config.yaml`